core/node/da_clients/src/eigen/client.rs

```diff
@@ -291,6 +300,7 @@ mod tests {
         assert_eq!(retrieved_data.unwrap(), data);
     }
 
+    #[ignore = "remote dependency"]
```

Comment:
replace it with something more descriptive like:

"Depends on external RPC node"

Reply:
Done!